google/cloud/iam/iam_credentials_connection.gcpcxx.pb.cc

```diff
@@ -63,36 +54,31 @@ IAMCredentialsConnection::SignJwt(
   return Status(StatusCode::kUnimplemented, "not implemented");
 }
 
-namespace {
-std::unique_ptr<IAMCredentialsRetryPolicy> DefaultRetryPolicy() {
+std::unique_ptr<IAMCredentialsRetryPolicy> DefaultIAMCredentialsRetryPolicy() {
   return IAMCredentialsLimitedTimeRetryPolicy(std::chrono::minutes(30)).clone();
 }
 
-std::unique_ptr<BackoffPolicy> DefaultBackoffPolicy() {
+std::unique_ptr<BackoffPolicy> DefaultIAMCredentialsBackoffPolicy() {
```

Comment:
These "Default" functions are not needed anymore. The body of them can just be inlined in the body of the "FooDefaultOptions()" function.

Fine to do in a followup PR if you prefer.